### DISCUSSION TITLE
`pnpm db:push` not working!? Needs generate -> migrate -> push?

### DISCUSSION THREAD

**Discussion by moritzWa** [t3-oss/create-t3-turbo#1046] [Q&A]
Is the push command not processing the changes to the schema.ts file for anyone else?

I added:
```
"generate": "pnpm with-env drizzle-kit generate",
"migrate": "pnpm with-env drizzle-kit migrate",
```
but it's a bit annoying running this (and the push command) every time I make schema change.

**Comment by trevorpfiz** [comment]
it should work? you either use db push with `drizzle-kit push`, or if you are using migrations, use `drizzle-kit generate` and `drizzle-kit migrate`. https://orm.drizzle.team/kit-docs/overview#prototyping-with-db-push
